common/courses/states.ts

```diff
@@ -12,6 +21,39 @@ export async function hasStarted(subcourse: Subcourse) {
     return startedLectures === 0;
 }
 
+
+export async function subcourseOverGracePeriod(subcourse: Subcourse) {
```

Comment:
What is this grace period for?

Reply:
After the course ended, participants might want to send feedback to the instructor or ask followup questions, instructors might want to send some final results or promote a next course. So we allow certain course actions for a 'grace period' (terrible name, couldn't come up with something better) Should I add a comment?

Reply:
Thank you for the explanation

> Should I add a comment?

That would be amazing.